package.json

```diff
@@ -21,9 +21,10 @@
   },
   "dependencies": {
     "@flatten-js/interval-tree": "^1.0.18",
-    "@fortawesome/fontawesome-free": "^5.15.4",
+    "@fortawesome/fontawesome-free": "^6.2.1",
```

Comment:
Are we ok bumping the version here? I found I needed to bump for `file-arrow-down`, even though it was added in 5.1.0.

Reply:
Yes, I dont see any breaking changes for us in fa 6. They dropped support for ie but we have done so already so all's good